docs/topics/proc_add-debug-logging-spring-boot.adoc

```diff
@@ -0,0 +1,48 @@
+= Add Debug Logging
+
+Add debug logging to your application.
+
+.Prerequisites 
+* An application you want to debug. For example, the xref:mission-http-api-spring-boot-tomcat[{name-mission-http-api} booster].
+
+.Procedure
+
+. Declare a `org.apache.commons.logging.Log` object using the `org.apache.commons.logging.LogFactory` for the class you want to add logging.
++
+[source,java,options="nowrap",subs="attributes+"]
+----
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+...
+private static Log logger = LogFactory.getLog(TheClass.class);
+----
++
+For example, if you wanted to add logging to the `GreetingEndpoint` class in the xref:mission-http-api-spring-boot-tomcat[{name-mission-http-api} booster], you would use `GreetingEndpoint.class`.
+
+. Add debugging statements using `logger.debug("my logging message")`.
++
+.Example logging statement
+[source,java,options="nowrap",subs="attributes+"]
+----
+@GET
+@Path("/greeting")
+@Produces("application/json")
+public Greeting greeting(@QueryParam("name") @DefaultValue("World") String name) {
+    String message = String.format(properties.getMessage(), name);
+    
+    logger.debug("Message: " + message);
+    
+    return new Greeting(message);
+}
+----
+
+. Add a `logging.level.full.qualified.name.of.TheClass=DEBUG` in `src/resources/application.properties`.
```

Comment:
It's not "full", it's "fully". _Fully qualified name_ is actually a technical term: https://en.wikipedia.org/wiki/Fully_qualified_name